src/components/MultipleFileInputField/MultipleFileInputField.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form, FormControl, IconButton,
+} from '@edx/paragon';
+import classNames from 'classnames';
+import * as FontAwesome from '@fortawesome/free-solid-svg-icons/faTimes';
+import { getSizeInBytes, formatBytes } from './utils';
+
+const MultipleFileInputField = ({
+  input,
+  label,
+  type,
+  description,
+  disabled,
+  required,
+  className,
+  id,
+  meta: { touched, error },
+  ...props
+}) => {
+  const hasError = !!(touched && error);
+  const [size, setSize] = useState('0');
```

Comment:
not actually it is a string because I am showing the bytes, kb, mb metric with the number. 
So It is a string initially and then when first file is uploaded its byte size is converted into most appropriate unit and then when later file is added previous string with a unit is converted into bytes according to unit and then bytes are summed to calculate total size. After that total bytes are again reformatted into string containing units as well. this logic is in utils.js in same dir.